### PR TITLE
feature/ads-629 Control class naming

### PR DESCRIPTION
**Pull request by APJohns**

### CODE REVIEW

File: src/library/styles/components/_input-radio.scss [hunk 1]
```diff
@@ -1,21 +1,21 @@
...
-  > .bsds-input {
+  > .bsds-field {
     &:not(:first-child) {
-      .bsds-input-radio {
+      .bsds-input-radio-control {
         margin-top: space(2x);
       }
     }
   }
 }
```
Comment (APJohns) [inline src/library/styles/components/_input-radio.scss:16]: Do we want to abstract this to a shared place since the spacing/code is the same as checkbox?

Comment (APJohns) [reply src/library/styles/components/_input-radio.scss:16]: Do we want to do the same with focus and invalid? Anything else?

File: src/library/styles/components/_search.scss [hunk 2]
```diff
@@ -1,16 +1,17 @@
-.bsds-search {
-  width: 100%;
-
-  .bsds-input {
-    height: 100%;
-    margin-top: 0;
-  }
+.bsds-field-text-label {
+  margin-bottom: space(md);
```
Comment (kaseyllabelle) [inline src/library/styles/components/_search.scss:2]: can we move this to the top of something?

File: src/library/styles/components/_search.scss [hunk 3]
```diff
@@ -1,16 +1,17 @@
-.bsds-search {
-  width: 100%;
-
-  .bsds-input {
-    height: 100%;
-    margin-top: 0;
-  }
+.bsds-field-text-label {
```
Comment (kaseyllabelle) [inline src/library/styles/components/_search.scss:1]: initially thought this was used elsewhere but we have .bsds-field-label-text and .bsds-field-text-label
thoughts on standardizing or reconsidering naming?

Comment (kaseyllabelle) [reply src/library/styles/components/_search.scss:1]: actually, looks like this is getting visually hidden anyway. can we just remove?

**Review by kaseyllabelle — APPROVED** [review]

**Pull request merged**